regEx/challenges-04.test.js

```diff
@@ -0,0 +1,240 @@
+'use strict';
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 1
+//
+// Write a function named isNum that takes in a string or number of any length.
+// This function should use a regular expression pattern to return true if the input
+// contains a number, and false if the input does not contain a number.
+//
+// Example input/output:
+// 12345 => true
+// '12345' => true
+// 'h3llo world' => true
+// 'hello world' => false
+// ------------------------------------------------------------------------------------------------
+
+const isNum = (num) => {
+    let regex = /[0-9]/;
+    return regex.test(num);
+};
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 2
+//
+// Write a function named isCapitalized that takes in a string. This function should use a
+// regular expression pattern to match all words that begin with a capital letter.
+//
+// Return an array containing all the matches.
+// ------------------------------------------------------------------------------------------------
+
+const isCapitalized = (string) => {
+  let regex = /^[A-Z]/;
+  const capitalArr = [];
+  let newStr = string.split(' ');
+  newStr.forEach( (word) => {
+    if(regex.test(word)){
+      capitalArr.push(word);
+    }
+  });
+  return capitalArr;
+};
+
```

Comment:
can you do this using the match() method? That returns an array of matched elements so it will cut your code down significantly (to only 2-3 lines) as you won't have to split your string or create a new array to return